Samples/DynamicDependencies/Basic/cs-console-unpackaged/DynamicDependenciesBasicCS/DynamicDependenciesBasicCS/DynamicDependenciesBasicCS.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
```

Comment:
Should we add TargetPlatformMinVersion for 10.0.17763.0 here?

Reply:
there is a bug in preview2 that prevents targeting 17763.  We can make an update when its fixed in next release.